hangman.rb

```diff
@@ -0,0 +1,262 @@
+# Pair: Alice Rhomieux
+require 'colorize'
+
+class Hangman
+  def initialize
+    @game_word = pick_word
+    @wrong_letters = []
+    @errors = 0
+
+    # initializes array_of_slots with underscores
+    @num_of_slots = @game_word.length
+    @array_of_slots = []
+    @num_of_slots.times do
+      @array_of_slots.push("_ ")
+    end
+
+    @drawings = [[
```

Comment:
I think that these make sense to initialize in this method - but for readabilities sake it would make sense to return this array from a separate method